notebooks/data_obfuscation.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<p>Is there a PyPi implementation of this as well? If so, do you want to just mention that here like the other methods?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/operate-first-data-science-community/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Yes, I will add that up. Thanks :) </p>